Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -2851,16 +2851,6 @@ def gen2018HiMix(fragment,howMuch):
 
 steps['RECOFROMRECOSt2']=steps['RECOFROMRECO']
 
-steps['RECODFROMRAWRECO']=merge([{'-s':'RAW2DIGI:RawToDigi_noTk,L1Reco,RECO:reconstruction_noTracking,EI',
```

Comment:
@guitargeek always nice to have some clean up :) shall we also clean this up everywhere in relval code?
https://github.com/cms-sw/cmssw/blob/02d4198c0b6615287fd88e9a8ff650aea994412e/Configuration/PyReleaseValidation/python/relval_production.py#L17
https://github.com/cms-sw/cmssw/blob/c1f0db57b8464dfb2388ed74ec25d80df9b6abf7/Configuration/PyReleaseValidation/python/MatrixReader.py#L341
https://github.com/cms-sw/cmssw/blob/c1f0db57b8464dfb2388ed74ec25d80df9b6abf7/Configuration/PyReleaseValidation/python/MatrixReader.py#L243

Reply:
Hi, Good to know you are ok with this! To whom did you direct your question to? I agree of course that this should be cleaned so nobody in the future is confused about this, but I think  as PdmV convenor it's your decision to make?

Reply:
I think @chayanit is proposing that you include the additional cleanup in this PR. I am also in favor of this.

Reply:
Yes you are right, so let's do that